tests/Kernel/ContactRenderTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
```

Comment:
Since we need to test entity view step by step I decided to implement Contact full view in the separate test.